src/Components/Header/Header.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import s from './Header.module.css'
+import {NavLink} from "react-router-dom";
+
+const Header = () => {
+    return <header className={s.header}>
+        <table style={{width:'100%'}}>
+            <th>
+                <div>
+                    My Jira
+                </div>
+            </th>
+            <th  style={{textAlign: 'right'}}>
+                <div>
+                    <a href="/login">Login</a>|<a href="/register">Register</a>
```

Comment:
I suggest to use `history.push` or `Link` component for changes pages, in this case, the whole page will not be rerendered (at the current moment on link click application reloads html, js, css like on the first open)

Reply:
replaced with navlink